components/usage/pkg/apiv1/usage.go

```diff
@@ -136,6 +136,20 @@ func (s *UsageService) ReconcileUsage(ctx context.Context, req *v1.ReconcileUsag
 
 }
 
+func (s *UsageService) GetCostCenter(ctx context.Context, in *v1.GetCostCenterRequest) (*v1.GetCostCenterResponse, error) {
+	result, err := db.GetCostCenter(ctx, s.conn, db.AttributionID(in.AttributionId))
```

Comment:
It's always worth checking the input you're getting. In this case, I think you can do the following:
1. Check `in.AttributionId` is not empty
2. Do `db.ParseAttributionID` and ensure it doesn't return an error